### DISCUSSION TITLE
Security Onion Dashboards

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5482] [Unsupported Versions]
Hello everyone! Just curious but are there any  Kibana pros that are making custom dashboards specifically around the MITRE Attack framework? If so I would love to help.

**Comment by weslambert** [comment]
Hi. @ColeVan , a while back someone shared some for a previous version of Security Onion.  Maybe you can draw inspiration from those?

https://networkforensic.dk/SecurityOnion/default.html
